lib/hyperion/formats.rb

```diff
@@ -18,6 +18,7 @@ def write(obj, format)
       case Formats.get_from(format)
       when :json; write_json(obj)
       when :protobuf; obj
+      when :form_data; obj
```

Comment:
Despite the fact that typhoeus doesn't let us specify per-part content types, I still would like to tweak this a little: json and protobuf specify the _encoding_, whereas form_data specifies the _structure_. So I would be in favor of instead creating a `Multipart` struct that contains a single hash which is the multipart body. Then have this `write` method handle a `Multipart` `obj` specially.